# Remove a motion

string motion_key
---
bool success